src/components/Markdown/Headings.tsx

```diff
@@ -52,7 +51,6 @@ export const Headings: FunctionComponent<HeadingResolverProps> = ({
         to={`#${id}`}
       >
         {children}
-        {level < 3 && <Hr />}
```

Comment:
What's this?

Reply:
I noticed whenever user markdown already has dividers between headers, this would create an extra one. IMO it'll be better if we just update the API reference markdown to place dividers as needed